DS_1/HighestDegree.java

```diff
@@ -0,0 +1,74 @@
+package datastructure_1;
+import java.util.ArrayList;
+import java.util.Scanner;
+/**
+ * HighestDegree class to find the highest degree of the polynomial variant using the nested list implementation
+ *
+ */
+public class HighestDegree {
+/**
+ * this method create polynomial expression in nested list form
+ * @return Polynomial expression in form of nested list
+ */
+	public ArrayList<ArrayList<String>> createPolynomial()
```

Comment:
No need to make method public, if the method is to be used within this file only.